indico/web/forms/widgets.py

```diff
@@ -80,12 +80,13 @@ class JinjaWidget(object):
                           separate kwargs.
     """
 
-    def __init__(self, template, plugin=None, single_line=False, single_kwargs=False, **context):
+    def __init__(self, template, plugin=None, single_line=False, single_kwargs=False, inline_js=False, **context):
         self.template = template
         self.plugin = plugin
         self.context = context
         self.single_line = single_line
         self.single_kwargs = single_kwargs
+        self.inline_js = inline_js
```

Comment:
Yeah, but then I need to extract from `jsonify_form` the bit that gets the JS from the form. Otherwise how will I handle that server-side? Plus, at least this way the old `<script>` tags go away.